final_prep/README.md

```diff
@@ -19,37 +19,42 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+<img width="1163" alt="Screen Shot 2021-06-14 at 6 05 10 PM" src="https://user-images.githubusercontent.com/83977143/121973936-91ee2500-cd3b-11eb-9472-8ecae3290ef7.png">
+
+
+<img width="1172" alt="Screen Shot 2021-06-14 at 6 05 23 PM" src="https://user-images.githubusercontent.com/83977143/121973969-a8947c00-cd3b-11eb-97e1-2333c5c3dbd5.png">
+
+
+<img width="1135" alt="Screen Shot 2021-06-14 at 6 05 34 PM" src="https://user-images.githubusercontent.com/83977143/121973977-af22f380-cd3b-11eb-954b-59cd7372234f.png">
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/jpage13/3679a30ae84c5990099c1b9aea8251e5
```

Comment:
I like how you show personality in your letter to your mentor, but be sure to express excitement to meet them, and work together! You also may add a specific thing or 2 you are excited about in making this change.